itests/hcatalog-unit/src/test/java/org/apache/hive/hcatalog/listener/TestDbNotificationListener.java

```diff
@@ -1584,69 +1510,39 @@ public void sqlInsertPartition() throws Exception {
 
     event = rsp.getEvents().get(12);
     assertEquals(firstEventId + 13, event.getEventId());
-    assertEquals(EventType.ADD_PARTITION.toString(), event.getEventType());
+    assertEquals(EventType.INSERT.toString(), event.getEventType());
 
     event = rsp.getEvents().get(13);
     assertEquals(firstEventId + 14, event.getEventId());
-    assertEquals(EventType.INSERT.toString(), event.getEventType());
-    // Parse the message field
-    verifyInsert(event, null, tblName);
+    assertEquals(EventType.ALTER_PARTITION.toString(), event.getEventType());
```

Comment:
What is the reason of those even type changes? It looks like now the test does a whole different thing.

Reply:
There are only 29 total events generated and which are correct based on sql commands used in tests. With UPDATE_PARTITION_COLUMN_STAT event event flow is different. So I have tested assertion for total 29 events and removed unnecessary code

Reply:
I have the same question here: I don't understand why the event types changed in the test. Why those are necessary?

Reply:
fixed. Inert in table with partition generates different events based on command used
 insert into table mytb partition (ds) values (2, 'today')- generates 1 ALTER_PARTITION events
insert into table mytb partition (ds = 'today') values (2) - generates 2 ALTER_PARTITION events

Reply:
thx